src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
```

Comment:
Consider adding a semicolon at the end of the interface declaration for consistency and to avoid errors. 🚫